README.md

```diff
@@ -33,67 +32,70 @@ This off-chain application comprises 7 main goroutines: Monitor, Verifier, Vote
 ## Deployment
 
 ### Config
-1. Set your private key import method (via file or aws secret), deployment environment and gas limit.
-```
-  "greenfield_config": {
-    "key_type": "local_private_key" or "aws_private_key" depending on whether you are storing the keys on aws or locally in this json file
-    "aws_region": set this if you chose "aws_private_key"
-    "aws_secret_name": set this if you chose "aws_private_key"
-    "aws_bls_secret_name": set this if you chose "aws_private_key"
-    "private_key": set this if you chose "local_private_key"
-    "bls_private_key": set this if you chose "local_private_key" 
-    "rpc_addrs": [
-      "http://0.0.0.0:26750"
-    ],
-    "chain_id_string": chain id of the network, e.g., "greenfield_9000-121"
-    "gas_limit": transaction gas limit, e.g., 1000,
-    "fee_amount": transaction fees, e.g., "5000000000000",
-    "fee_denom": transaction fees denom, e.g., "BNB",
-    "no_simulate": simulate transaction, e.g., true
-    "deduplication_interval": skip events that were recently processed, e.g., 100
-  }
-```
 
-2. Set your log and backup preferences.
-```
-"log_config": {
-  "level": "DEBUG",
-  "filename": "log.txt",
-  "max_file_size_in_mb": 100 (file size threshold)  
-  "max_backups_of_log_files": 2 (backup count threshold)
-  "max_age_to_retain_log_files_in_days": 10 (backup age threshold)
-  "use_console_logger": true,
-  "use_file_logger": false,
-  "compress": false
-}
-```
-
-4. Config your database settings.
-```
-"db_config": {
-  "dialect": "mysql",
-  "db_path": "your_db_path"
-  "key_type": "local_private_key" or "aws_private_key" depending on whether you are storing the keys on aws or locally in this json file
-  "aws_region": set this if you chose "aws_private_key"
-  "aws_secret_name": set this if you chose "aws_private_key"
-  "username": set this if you chose "local_private_key"
-  "password": set this if you chose "local_private_key"
-  "max_idle_conns": 20, (set according to your db performance)
-  "max_open_conns": 40, (set according to your db performance)
-  "debug_mode": false  
-}
-```
+1. Set your private key import method (via file or aws secret), deployment environment and gas limit.
```

Comment:
can you guide user to refer to `https://github.com/bnb-chain/bnb-chain-charts/blob/d09a93d7b18e0f2afb537a23276a95c877e070b6/gnfd-challenger-testnet-values/values.yaml#L8` as well? 
As it is the testnet config.

Reply:
Done